third_party/move/move-model/bytecode/src/function_target.rs

```diff
@@ -172,6 +172,11 @@ impl<'env> FunctionTarget<'env> {
         self.func_env.is_mutating()
     }
 
+    /// Returns the type parameters of this function.
+    pub fn get_type_parameters(&self) -> Vec<TypeParameter> {
+        self.func_env.get_type_parameters()
+    }
+
     /// Returns the number of type parameters associated with this function, this includes both
     /// the defined type parameters and the ghost type parameters.
     ///
```

Comment:
Not related to this PR but making a comment as found it while reading code here: we should use `get_type_parameter_count()` below instead of `get_type_parameters().len()` to avoid cloning the type params just to get the length.

Reply:
+1